### DISCUSSION TITLE
Is there a quick way to update a pangenome for a newly available genome?

### DISCUSSION THREAD

**Discussion by pliang64** [ComparativeGenomicsToolkit/cactus#1549] [Q&A]
Dear all: I'm new to the use of pangenome, so this question may be very naive. It seems that the process of generating a pangenome graph from scratch based on a large number of genomes is very resource demanding. So I wonder if there is a quick way to keep updating an existing pangenome graph for newly available genomes, meaning we only to align the new genome (s) to the pangenome and update the pangenome graph with newly identified variants. If this is possible, please share the procedure to use. I thank you for your input on this in advance.

**Comment by glennhickey** [comment]
minigraph supports this but minigraph-cactus does not.

**Reply by pliang64** [reply]
Thanks for the information. So I guess with minigraph-cactus, we'd have to add the new genome to the list of old genomes and regenerate a pangenome graph, right? 

With minigraph, I guess one can do something like below?
minigraph -cxggs  pg_old.gfa newgenome.fa > pg_new.gfa

Anything useful with what "vg augment" can do? I know this utility is still experimental and doesn't work for SVs.
